### ISSUE TITLE
Quest: The Darkscale Warlord

### ISSUE THREAD

**Issue by zwei7** [HellscreamWoW/Tracker#3282]
I killed the Boss Naga named Wrathspine, but my quest log won't trigger that he is dead. Below I am looking at his corpse and my log says it is not ready to turn in. This Naga did not sparkle to indicate he has loot either.
![wowscrnshot_102815_145542](https://cloud.githubusercontent.com/assets/15259298/10799854/675eac1c-7d85-11e5-9df8-b3bfa8c455ab.jpg)

**Comment by silverjade** [comment]
Yes, the quest is supposed to auto-complete upon killing the mob and provide the follow-up quest. http://www.wowhead.com/quest=13899/the-darkscale-warlord (http://www.wowhead.com/quest=13900/the-offering-to-azshara is the follow-up)

**Comment by Ardente47** [comment]
Confirmed
